<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewInvoiceDlg</class>
 <widget class="QDialog" name="NewInvoiceDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New invoice</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/Icons.qrc">
    <normaloff>:/Icons/document-new.png</normaloff>:/Icons/document-new.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="TypeBox">
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Purchase</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sale</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cash purchase</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cash sale</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Save withdrawal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Save deposit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cash withdrawal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cash deposit</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Relation:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Invoice date:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateEdit" name="InvoiceDateEdit">
       <property name="displayFormat">
        <string>dd-MM-yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Invoicenumber:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="InvoiceNumberEdit">
       <property name="maxLength">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Grossamount:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="GrossamountSpin">
       <property name="prefix">
        <string>€ </string>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>VAT%:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="VATBox">
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>21</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Extra costs:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDoubleSpinBox" name="ExtraCostSpin">
       <property name="prefix">
        <string>€ </string>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Payed at:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QDateEdit" name="PayedDateEdit">
       <property name="displayFormat">
        <string>dd-MM-yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Comment:</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QTextEdit" name="CommentEdit">
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="RelationBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="IDEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="VATCheck">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>VAT amount:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QCheckBox" name="NettCheck">
       <property name="text">
        <string>Nettamount:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDoubleSpinBox" name="NettamountSpin">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="prefix">
        <string>€ </string>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="VATamountSpin">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="prefix">
        <string>€ </string>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>VAT year:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>VAT period:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QComboBox" name="VATPeriodBox">
       <item>
        <property name="text">
         <string>Q1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Q2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Q3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Q4</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QDateEdit" name="VATYearEdit">
       <property name="displayFormat">
        <string>yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>257</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="SaveButton">
     <property name="text">
      <string>Save</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>TypeBox</tabstop>
  <tabstop>RelationBox</tabstop>
  <tabstop>InvoiceDateEdit</tabstop>
  <tabstop>InvoiceNumberEdit</tabstop>
  <tabstop>GrossamountSpin</tabstop>
  <tabstop>VATBox</tabstop>
  <tabstop>VATCheck</tabstop>
  <tabstop>VATamountSpin</tabstop>
  <tabstop>NettCheck</tabstop>
  <tabstop>NettamountSpin</tabstop>
  <tabstop>ExtraCostSpin</tabstop>
  <tabstop>PayedDateEdit</tabstop>
  <tabstop>CommentEdit</tabstop>
  <tabstop>SaveButton</tabstop>
  <tabstop>IDEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../Resources/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
